packages/graphiql/README.md

```diff
@@ -127,44 +175,107 @@ GraphiQL supports customization in UI and behavior by accepting React props and
+        url: 'https://my.graphql.service/graphql',
+        headers: {
+          Authorization: "Bearer " + token
+        }
+      })
+    })
+  </script>
+</body>
+
+</html>
+
+```
+
+If you don't supply a custom `fetcher` function, but you do supply a `url`, then it will try to `POST` a standard GraphQL HTTP request to that URL, as well as retrieve the introspection schema from that URL.
+
+We reccomend using `unpkg` over cdnjs, as does react. The updates to their CDN are almost instantaneous. GraphiQL 0.x.y will be undergoing a lot of changes and releases before we get to a stable version.
+
+#### CDN Bundle: graphiql.min.js
+
+The `graphiql.min.js` continues to work as always. Here's how to acheive the same implementation as above.
+Note that the second argument to `React.createElement` are [the props for GraphiQL](#graphiql-props).
+
+```js
+<div id="graphiql">Loading...</div>
+<script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+<script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+<script type="text/javascript" src="https://unpkg.com/browse/graphiql@latest/graphiql.min.js"></script>
+<script>
+  const getFetcher = token => async (graphQLParams) => {
+   const result = await fetch('https://my.graphql.service/graphql', { 
+     headers: { 
+       Authorization: "Bearer " + token, 
+       Content-type: 'application/json'
+     }, 
+     method: "POST"
+    })
+    return result.json()
+  }
+  loginToMyService().then(token => {
+    ReactDOM.render(
+      React.createElement(GraphiQL, {
+        fetcher: getFetcher(token)
+      }),
+      document.getElementById('graphiql')
+    );
+  }
+</script>
+```
+
 ### Usage Examples
 
 ```js
+GraphiQL.Logo = <div>Custom Logo</div>
+
+
 class CustomGraphiQL extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       // REQUIRED:
       // `fetcher` must be provided in order for GraphiQL to operate
-      fetcher: this.props.fetcher,
-
-      // OPTIONAL PARAMETERS
-      // GraphQL artifacts
-      query: '',
-      variables: '',
-      response: '',
-
-      // GraphQL Schema
-      // If `undefined` is provided, an introspection query is executed
-      // using the fetcher.
-      schema: undefined,
-
-      // Useful to determine which operation to run
-      // when there are multiple of them.
-      operationName: null,
-      storage: null,
-      defaultQuery: null,
-
-      // Custom Event Handlers
-      onEditQuery: null,
-      onEditVariables: null,
-      onEditOperationName: null,
-
-      // GraphiQL automatically fills in leaf nodes when the query
-      // does not provide them. Change this if your GraphQL Definitions
-      // should behave differently than what's defined here:
-      // (https://github.com/graphql/graphiql/blob/master/src/utility/fillLeafs.js#L75)
-      getDefaultFieldNames: null
+      fetcher: this.props.fetcher
```

Comment:
Are these state fields documented elsewhere that's easy to copy/paste?

If fetcher is on props then it doesn't really need state; so this whole example doesn't need state?